test-boxed-cpp.cpp

```diff
@@ -53,8 +53,9 @@ TEST_CASE("boxed_cast with different inner types")
 {
     auto constexpr a = N{3};
     auto constexpr b = boxed_cast<Z>(a);
-
+#ifndef __GNUG__
```

Comment:
why is this only working for GNU C++? 🤔 hmmmm

EDIT: oh wait. for all but GNU C++. Okay, all good. :D